decentraland/components/presentational/AssetName.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { StyleSheet, View, Text } from "react-native";
+import PropTypes from "prop-types";
+import { responsiveHeight } from "react-native-responsive-dimensions";
+import Colors from "@constants/Colors";
+
+export function AssetName({ asset }) {
+  let { name } = asset;
+  if (!name) name = "(No name for this one right now)";
+
+  return (
+    <View style={styles.landNameContainer}>
+      <Text style={styles.landName}>{name}</Text>
```

Comment:
Why not just pass the name in if that's all this component uses?

Reply:
Usually, I pass the entire object on situations like that because other info could be used in future (e.g. Shows an icon if is estate and another if is parcel). 
But since that component is too specific, it doesn't make sense. Just changed.

Reply:
Makes sense to me, I'll keep that in mind. Thanks for changing it in this specific case. Resolving this.